FCCee/IDEA/compact/IDEA_o1_v03/DriftChamber_o1_v02.xml

```diff
@@ -0,0 +1,137 @@
+  <constant name="DCH_ncell"                value=" 192 "    />
+  <constant name="DCH_ncell_increment"      value=" 48 "    />
+  <constant name="DCH_nsuperlayers"         value=" 14 "    />
+  <constant name="DCH_nlayersPerSuperlayer" value=" 8 "    />
+  <constant name="DCH_ncell_per_sector"     value=" 24 "   />
+
+  <!-- Alfa = twistangle/2   -->
+  <constant name="DCH_alpha"                value=" 15*deg "    />
+
+  <!--  Parameters of first layer  -->
+  <constant name="DCH_first_sense_r"        value=" DCH_guard_inner_r_at_z0 + 8*mm "    />
+  <constant name="DCH_first_width"          value=" 2*pi* DCH_first_sense_r / DCH_ncell"    />
+
+  <!--
+  Details about geometry of wires:
+    - guard wire:             50 um Al (core), 0.3 um Ag (coating)
+    - sense wire:             20 um W  (core), 0.3 um Au (coating)
+    - field wires top/bottom: 40 um Al (core), 0.3 um Ag (coating)
+    - field wire center:      50 um Al (core), 0.3 um Ag (coating)
+  -->
+  <!-- sense wire thickness (total)   -->
+  <constant name="DCH_SWire_thickness"           value="0.0203*mm"   />
+
+  <!-- Field Side (top/bottom) wire thickness (total)   -->
+  <constant name="DCH_FSideWire_thickness"       value="0.0403*mm"   />
+
+  <!-- Field Central wire thickness (total)   -->
+  <constant name="DCH_FCentralWire_thickness"    value="0.0503*mm"   />
+
+  </define>
+
+  <limits>
+    <limitset name="DCH_limits">
+      <limit name="step_length_max" particles="e[+-]" value="1.0" unit="mm" />
+      <limit name="step_length_max" particles="mu[+-]" value="2.0" unit="mm" />
+      <limit name="step_length_max" particles="*" value="1.0" unit="mm" />
+    </limitset>
+  </limits>
+  <regions>
+    <region name="DCH_region" eunit="eV" lunit="mm" cut="1.0" threshold="1.0">
+      <limitsetref name="DCH_limits"/>
+    </region>
+  </regions>
+
+  <detectors>
+    <detector
+      id="DetID_DCH"
+      name="DCH_o2"
+      type="DriftChamber_o1_v02_T"
+      readout="DCHCollection"
+      region="DCH_region"
+      limits="DCH_limits"
+      printExcelTable="True"
+      >
+    <!-- /detectors/detector/vessel -->
+    <vessel
+        material="CarbonFibStr"
+        vis="dch_vessel_vis"
+      >
+    </vessel>
+    <!-- /detectors/detector/gas -->
+    <gas
+        material="GasHe_90Isob_10"
+        vis="dch_gas_vis"
+      >
+    </gas>
+    <!-- /detectors/detector/wires -->
+    <wires
+        vis="dch_no_vis_nodaughters"
+        buildSenseWires="True"
+        buildFieldWires="True"
+        SWire_thickness       ="DCH_SWire_thickness"
+        FSideWire_thickness   ="DCH_FSideWire_thickness"
+        FCentralWire_thickness="DCH_FCentralWire_thickness"
+        SWire_material       ="W"
+        FSideWire_material   ="Al"
+        FCentralWire_material="Al"
+      >
+    </wires>
+    </detector>
+  </detectors>
+
+  <readouts>
+    <readout name="DCHCollection">
+      <id>system:5,superlayer:4,layer:7,nphi:15,stereosign:3</id>
```

Comment:
Proposal as discussed: no redundancy between superLayer and layer (superLayer from 0 to 13, layer from 0 to 7 in the present design). Let's still have one additional safety-bit in case this changes: 5 bits for  super layer, 4 bits for layer. For phi, the maximum number of cell (for the last superlayer) will be 192+13*48=816. I propose therefore 11 bits for this field. Stereo could be 2 bits, or "-1"